Resources/Prototypes/Entities/Constructible/Specific/Engines/Singularity/emitter.yml

```diff
@@ -1,6 +1,4 @@
 ﻿- type: entity
-  name: emitter
-  description: "A machine that fires bolts of energy, used for powering containment fields at a safe distance."
```

Comment:
why remove name & desc?

Reply:
fluent ocalization adds the automatically

Reply:
or it doesnt right now becaeuse its broken cool. ill leave it in